### PR TITLE
support ernie trt-int8 for inference

### PR DESCRIPTION
**Pull request by ceci3**
### PR types

New features

### PR changes

Others

### Describe

support  trt-int8 inference for ernie
通过load_inference_model fake 量化测试精度结果为0.7786

layer  | prec | prec
-- | -- | --
skip ln | FP16 | FP16
qkv2ctx | FP16 | FP16
fc | FP16 | int8
latency(bs=40，T4) | 35.5341ms | 29.1797ms
acc | 0.7786 | 0.7770
qps |1898seq/s| 2310seq/s

### CODE REVIEW

File: paddle/fluid/framework/ir/multihead_matmul_fuse_pass.cc [hunk 1]
```diff
@@ -535,6 +535,67 @@ static int BuildFusionV2(Graph* graph, const std::string& name_scope,
...
+            BOOST_GET_CONST(float, mul1_op_desc->GetAttr("out_threshold"));
+        auto out_scale2 =
+            BOOST_GET_CONST(float, mul2_op_desc->GetAttr("out_threshold"));
+        auto out_scale_max = std::max(out_scale0, out_scale1);
+        out_scale_max = std::max(out_scale_max, out_scale2);
+        multihead_op_desc.SetAttr("out_threshold", out_scale_max);
+      }
+    }
+
+    bool enable_int8 = mul0_op_desc->HasAttr("enable_int8");
```
Comment (cryoco) [inline paddle/fluid/framework/ir/multihead_matmul_fuse_pass.cc]: 这里是不是和上面重复了

File: paddle/fluid/inference/tensorrt/convert/fc_op.cc [hunk 2]
```diff
@@ -106,8 +106,19 @@ class FcOpConverter : public OpConverter {
     auto regist_fc = [&](nvinfer1::ITensor* inputs, int n_output,
                          TensorRTEngine::Weight& weight,
                          TensorRTEngine::Weight& bias) {
-      auto* fc_layer = TRT_ENGINE_ADD_LAYER(engine_, FullyConnected, *inputs,
-                                            n_output, weight.get(), bias.get());
+      nvinfer1::ILayer* fc_layer = nullptr;
+      if (enable_int8) {
+        CHECK(op_desc.HasAttr("out_threshold"));
```
Comment (cryoco) [inline paddle/fluid/inference/tensorrt/convert/fc_op.cc]: 用PADDLE_ENFORCE吧，给出报错信息

File: paddle/fluid/framework/ir/multihead_matmul_fuse_pass.cc (hunk 1 above)
Comment (ceci3) [reply paddle/fluid/framework/ir/multihead_matmul_fuse_pass.cc]: 删掉了，感谢~

File: paddle/fluid/inference/tensorrt/convert/fc_op.cc (hunk 2 above)
Comment (ceci3) [reply paddle/fluid/inference/tensorrt/convert/fc_op.cc]: 已修改，感谢~

**Review by cryoco — APPROVED** [review]
LGTM

**Review by chalsliu — APPROVED** [review]

**Pull request merged**